### PR TITLE
deprecate batch

### PR DESCRIPTION
**Pull request by bhakiyakalimuthu**
- deprecate handling batch requests
- disable go vet and static check for unreachable code

### CODE REVIEW

File: server/request_handler.go [hunk 1]
```diff
@@ -101,6 +102,13 @@ func (r *RpcRequestHandler) process() {
 	// Parse JSON RPC payload
 	var jsonReq *types.JsonRpcRequest
 	if err = json.Unmarshal(body, &jsonReq); err != nil {
+		r.requestRecord.UpdateRequestEntry(r.req, http.StatusBadRequest, err.Error())
```
Comment (metachris) [inline server/request_handler.go:105]: every single batch request will show up as error in the database. is this actually helpful?

File: server/request_handler.go [hunk 2]
```diff
@@ -101,6 +102,13 @@ func (r *RpcRequestHandler) process() {
...
+		r.logger.Warn("[process] Parse payload", "error", err)
+		(*r.respw).WriteHeader(http.StatusBadRequest)
+		return
+
+		// Deprecate handling batch - DO NOT PROCEED
+		// Note: go vet checks are disabled for unreachable code
```
Comment (metachris) [inline server/request_handler.go:111]: you could keep the linter and comment the code. but not important i think :)

File: server/request_handler.go (hunk 1 above)
Comment (bhakiyakalimuthu) [reply server/request_handler.go:105]: lets remove for now

File: server/request_handler.go (hunk 2 above)
Comment (bhakiyakalimuthu) [reply server/request_handler.go:110]: There are lot of code need to be commented.Linter does't allow. better to suppress linter warnings

**Review by metachris — APPROVED** [review]

**Pull request merged**